modules/apps/layout/layout-reports-web/src/main/java/com/liferay/layout/reports/web/internal/display/context/LayoutReportsDisplayContext.java

```diff
@@ -80,7 +80,7 @@ public LayoutReportsDisplayContext(
 			WebKeys.THEME_DISPLAY);
 	}
 
-	public Map<String, Object> getData() {
+	public Map<String, Object> getData() throws PortalException {
```

Comment:
Better if we don't trow the exception, I mean, if the back URL is broken is OK, we can handle it, we should not stop rendering the React component.

Reply:
I think we have a method called _getCompleteURL, that call this method and if yoiu find an exception returns the currentCompleteURL, what do you think about using it?

Reply:
Valid point, but see https://github.com/liferay-tango/liferay-portal/pull/1056#issuecomment-815031181. If for some reason we rollback to this previous solution I will do this.